CONTRIBUTING.md

```diff
@@ -24,12 +25,13 @@ We're always looking to make Akita better, so definitely drop us a Feature Reque
 if you have any ideas brewing!
 
 Issue Templates you can choose from:
+
 - 📄 [Create New Blank Issue](https://github.com/esse-dev/akita/issues/new?assignees=&labels=&template=blank-issue.md&title=)
 - 🐞 [Create New Bug Report](https://github.com/esse-dev/akita/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D)
 - 💡 [Create New Feature Request](https://github.com/esse-dev/akita/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%5BFEATURE+REQUEST%5D)
 
 ## Contribute Code
-	 
```

Comment:
I think it'd be good to have a line in here to read the Privacy Policy before contributing code (so developers understand that we want to uphold privacy!) and so they'll know to update the Privacy Policy if any code changes require updating the Privacy Policy